### PR TITLE
CIV-11542 Bilingual claim creation

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-11542

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -3242,6 +3252,7 @@
     "SECURITY_CODE_WRONG_OR_EXPIRED": "Nid oedd y cod diogelwch yn iawn neu mae wedi dod i ben",
     "SELECT_ADDITIONAL_TIME": "Dewiswch faint o amser ychwanegol yr ydych yn gwneud cais amdano",
     "SELECT_WELSH_AND_ENGLISH_OPTION": "Dewiswch Cymraeg os ydych am ymateb i'r hawliad hwn yn Gymraeg",
+    "CLAIM_SELECT_WELSH_AND_ENGLISH_OPTION": "Select Welsh if you want to create this claim in Welsh",
```
Comment (portizdev) [inline src/main/modules/i18n/locales/cy.json:3255]: Pls alphabetical order

**Review by portizdev — COMMENTED** [review]
I think it would be a good task to show & tell the developers and would help in the resolution of the next tasks.

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (ElifDenizEsea) [reply src/main/modules/i18n/locales/cy.json]: done

**Review by manish14836 — APPROVED** [review]

File: src/main/common/models/claim.ts [hunk 2]
```diff
@@ -105,6 +105,7 @@ export class Claim {
   directionQuestionnaire?: DirectionQuestionnaire;
   respondent1ResponseDate?: Date;
   claimBilingualLanguagePreference: ClaimBilingualLanguagePreference;
```
Comment (neeta-hmcts) [inline src/main/common/models/claim.ts:107]: just curious does claimBilingualLanguagePreference and claimantBilingualLanguagePrefrence not the same ??

Comment (ElifDenizEsea) [reply src/main/common/models/claim.ts:107]: claim coming from defendant

**Review by neeta-hmcts — APPROVED** [review]

**Review by jeswanth-hmcts — APPROVED** [review]

**Pull request merged**